### PR TITLE
Auto swagger

### PR DESCRIPTION
**Pull request by kolesnikdi**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,5 +1,5 @@
 SECRET_KEY=django-insecure-$pso2)*$i!8k#k_y^6j_31c@6@fv66+5)m1@0o3p3ii#p*use3
-auth_password=pNyBVB8lOlC4CfM5
-auth_user=segareta@ukr.net
+auth_password=Y32RepfABOJMYyui
```
Comment (ArtemKhriapa) [inline .env:2]: email_token

File: Web_Menu_DA/settings.py [hunk 2]
```diff
@@ -21,7 +24,8 @@
 ALLOWED_HOSTS = [
     '127.0.0.1',
 ]
-
+LOGIN_REDIRECT_URL = 'http://127.0.0.1:8000/user'
+LOGOUT_REDIRECT_URL = 'http://127.0.0.1:8000/django_auth/login'
```
Comment (ArtemKhriapa) [inline Web_Menu_DA/settings.py:28]: remove it

File: .env (hunk 1 above)
Comment (kolesnikdi) [reply .env:2]: done

File: Web_Menu_DA/settings.py (hunk 2 above)
Comment (kolesnikdi) [reply Web_Menu_DA/settings.py:28]: done

**Review by ArtemKhriapa — APPROVED** [review]

**Pull request merged**